src/Altinn.Notifications/Configuration/AuthorizationConstants.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Altinn.Notifications.Configuration
+{
+    /// <summary>
+    /// Constants related to authorization of notifications
+    /// </summary>
+    public static class AuthorizationConstants
+    {
+        /// <summary>
+        /// Id for the policy requiring create scope or access platform access token
+        /// </summary>
+        public const string POLICY_CREATE_SCOPE_OR_PLATFORM_ACCESS = "CreateScopeOrPlatfomAccessToken";
+
+        /// <summary>
+        /// Scope for allowing access to creating notifications
+        /// </summary>
+        public const string SCOPE_NOTIFICATIONS_CREATE = "altinn:notifications.create";
```

Comment:
We talked about using `altinn:serviceowner/notifications.create`, but I'm not sure.
Should we hold the door open for access by other parties?